### PR TITLE
first release of ocplib-simplex

### PR DESCRIPTION
**Pull request by OCamlPro-Iguernlala**
first release of ocplib-simplex

### CODE REVIEW

**Comment by OCamlPro-Iguernlala** [comment]
Is error 39 of opam linter described somewhere ?

**Comment by yallop** [comment]
It's referring to this:
```
["make"]
```

which invokes the command called `make` on the host system.  Since `make` can refer to different programs (GNU make, BSD make, etc.) on different systems, it's better to write

```
[make]
```
and let OPAM invoke the appropriate command for the system instead.

File: packages/ocplib-simplex/ocplib-simplex.0.1/descr [hunk 1]
```diff
@@ -0,0 +1,7 @@
+A library implementing a simplex algorithm, in a functional style, for solving systems of linear inequalities.
```
Comment (avsm) [inline packages/ocplib-simplex/ocplib-simplex.0.1/descr]: convention is not to have a full stop at the end of the first line

**Comment by OCamlPro-Iguernlala** [comment]
Ok thanks. I fixed it.

File: packages/ocplib-simplex/ocplib-simplex.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+license: "GNU Public Licence version 3.0"
+homepage: "https://github.com/OCamlPro-Iguernlala/ocplib-simplex"
+
+bug-reports: "https://github.com/OCamlPro-Iguernlala/ocplib-simplex/issues"
+dev-repo: "https://github.com/OCamlPro-Iguernlala/ocplib-simplex.git"
+
+
+build:[
+        ["autoconf"]
```
Comment (avsm) [inline packages/ocplib-simplex/ocplib-simplex.0.1/opam]: if this is calling autoconf then a dependency on `conf-autoconf` is needed

File: packages/ocplib-simplex/ocplib-simplex.0.1/descr (hunk 1 above)
Comment (OCamlPro-Iguernlala) [inline packages/ocplib-simplex/ocplib-simplex.0.1/descr]: fixed

File: packages/ocplib-simplex/ocplib-simplex.0.1/opam (hunk 2 above)
Comment (OCamlPro-Iguernlala) [inline packages/ocplib-simplex/ocplib-simplex.0.1/opam]: conf-autoconf already appears as a build dep

**Comment by OCamlPro-Iguernlala** [comment]
It's bizarre that each commit triggers a Travis check without canceling older ones. It is possible to do something so that the latest commit cancels builds that have been launched so far ?

**Comment by OCamlPro-Iguernlala** [comment]
@yallop @avsm 

Is the PR ok ? or should I do more fixes ?

**Pull request merged**